src/adhocracy_frontend/checkcode_and_compile.cfg

```diff
@@ -135,7 +135,7 @@ html_check =
 
 [nodejs]
 recipe = gp.recipe.node
-version = 4.2.4
+version = 4.4.5
 scripts = node npm
```

Comment:
> Because it is the current LTS version of node.

i know, but to be on the safe side i would prefere  the  node version suggested by webdrive